trunk/3rdparty/st-srs/Makefile

```diff
@@ -191,6 +191,9 @@ endif
 #
 # make EXTRA_CFLAGS=-DDEBUG_STATS
 #
+# or cache the stack and reuse it:
+# make EXTRA_CFLAGS=-DMD_CACHE_STACK
```

Comment:
why not add configure args to bypass `MD_CACHE_STACK` to `st-srs`, like `MD_VALGRIND`?

Reply:
I want to not cache the stack by default, while this macro is used to enable the stack cache.